Sources/StreamChatUI/Common Views/CurrentChatUserAvatarView.swift

```diff
@@ -60,6 +61,8 @@ open class _CurrentChatUserAvatarView<ExtraData: ExtraDataTypes>: Control, UICon
     override open func setUpLayout() {
         super.setUpLayout()
 
+        defaultIntrinsicContentSize = .init(width: 32, height: 32)
+
```

Comment:
Hmm, this looks more correct but that means the order of the lifecycle methods is wrong. We should probably call them like this:
```swift
        setUp()
        setUpLayout()
        (self as! Self).applyDefaultAppearance()
        setUpAppearance()
        updateContent()

```

Reply:
Thanks, you ar right. Changed for all...